### ISSUE TITLE
Allow players to choose between playermodels

### ISSUE THREAD

**Issue by Beaglerush** [better-modernized-combat/bmod-client#116]
I know this might seem excessive for the 0.1 board but it's the kind of thing that gets put off and forgotten forever if we never list it. I want to offer proper representation for our players, as soon as possible; you should be able to feel comfortably represented as either masc, fem or n/b when you're looking at yourself at the dealers and in the bar.

The bar itself is its own problem because talking to NPCs will be Trent's voice, which I imagine is a lot more work to deal with than the model. But just the model swapping would be a big W if we could get that in quicker. My ideal for now would be a basic choice between two helmeted spacesuit people in the Liberty style, and one of the bodies is more clearly feminine. Failing that, just having one Liberty helmed spacesuit person without any clearly defining features one way or the other would be acceptable.
Just anything better than asking our fem and n/b playtesters to always look like Trent would be meaningful.

**Comment by IrateRedKite** [comment]
This is actually already in the backlog with #77, but given the amount of work involved for what we get I would say its firmly something we should not scope for 0.1. I would like to scope this for 0.2 and do it properly, especially given that we don't have an easy way to swap player models around just yet.

**Comment by Beaglerush** [comment]
Sounds good, I'm fine with it being in 0.2 as that's not too far away and I don't want to drag out the tail of 0.1 any more than already happening. No further than 0.2, though.

**Comment by IrateRedKite** [comment]
@Beaglerush I'm actually going to shift this to 0.2 as I want the change scoped. As far as I know you've already adjusted the default playermodel, so nothing more to do here until we get a system in to do it.

**Comment by Beaglerush** [comment]
Yeah cool, so leaving in the current changes and worrying about maybe making it chooseable in the future, right? No problems from me, sounds sensible, and this works for now.
